syntax_test_jq.jq

```diff
@@ -72,6 +80,44 @@ def keys($value; $value; test):
 
 {i:0} | while(.i < 10 ; .i += 1)
 #       ^^^^^ keyword.control.flow.jq
+#                ^ keyword.operator.comparison.jq
+#                     ^ punctuation.terminator.jq
+#                          ^ keyword.operator.arithmetic.jq
+#                           ^ keyword.operator.assignment.jq
 
 {i:0} | until(.i == 10 ; .i += 1)
-#       ^^^^^ keyword.control.flow.jq
\ No newline at end of file
+#       ^^^^^ keyword.control.flow.jq
+#                ^^ keyword.operator.comparison.jq
+
+true false null
+# ^^ constant.language.boolean.jq
+#    ^^^^^ constant.language.boolean.jq
+#          ^^^^ constant.language.null.jq
+
+  0x123
```

Comment:
This should not be here, or somehow marked as invalid. JQ does not support hex notation for numbers.

Reply:
removed :)